### PR TITLE
Publish cloud drafts

### PR DESCRIPTION
**Pull request by andriitserkovnyi**
## PR Description

Move docs from drafts to public folders

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/dev/the-docker-sdk/202212.0/deploy-file/deploy-file-reference-1.0.md [hunk 1]
```diff
@@ -136,6 +136,27 @@ environment: 'docker'
 
 ***
 
+### environment-configuration: secrets:
```
Comment (hatemzidi-pro) [inline docs/scos/dev/the-docker-sdk/202212.0/deploy-file/deploy-file-reference-1.0.md:139]: i'm not sure that this part must be published.

File: docs/scos/dev/the-docker-sdk/202212.0/deploy-file/deploy-file-reference-1.0.md [hunk 2]
```diff
@@ -136,6 +136,27 @@ environment: 'docker'
...
+
+Defines secrets and their configuration:
+
+* `environment-configuration: secrets: name:`—secret's name.
+* `environment-configuration: secrets: grant:`—users' permissions. `limited` provides read while `public` provides read and write permissions. The default one is `limited`.
+* `environment-configuration: secrets: bucket:`—defines what the secret is used for. Acceptable values are `app`, `scheduler`, `pipeline`, and `common`. `common` is the default value. Common secrets can be used by all the buckets.
+
+***
+
+### environment-configuration: params:
```
Comment (hatemzidi-pro) [inline docs/scos/dev/the-docker-sdk/202212.0/deploy-file/deploy-file-reference-1.0.md:149]: same ! it's not meant to be published

**Review by hatemzidi-pro — APPROVED** [review]
LGTM

File: docs/scos/dev/the-docker-sdk/202212.0/define-parameters-and-secrets.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ environment-configuration:
 | - | - |
 | name | Unique identifier of a secret or parameter. You will use it when defining its value in `.env.docker.local`. |
 | grant | Users' permissions. `limited` provides read while `public` provides read and write permissions. The default one is `limited`. |
-| bucket | Defines what the secret or parameter is used for. Acceptable values are `app`, `scheduler`, `pipeline`, and `common`. `common` is the default value. Common secrets and parameters can be used by all the buckets. |
+| bucket | Defines what the secret or parameter is used for. Acceptable value is `app`. In future, we will enable variables to be injected into other components. |
```
Comment (VadymSachenko) [inline docs/scos/dev/the-docker-sdk/202212.0/define-parameters-and-secrets.md:32]: Predicting the future is not a good idea.
```suggestion
| bucket | Defines what the secret or parameter is used for. Acceptable value is `app`. |
```

File: docs/scos/dev/the-docker-sdk/202212.0/deploy-file/deploy-file-reference-1.0.md [hunk 4]
```diff
@@ -136,6 +136,27 @@ environment: 'docker'
 
 ***
 
+### environment-configuration: secrets:
+
+Defines secrets and their configuration:
+
+* `environment-configuration: secrets: name:`—secret's name.
+* `environment-configuration: secrets: grant:`—users' permissions. `limited` provides read while `public` provides read and write permissions. The default one is `limited`.
```
Comment (VadymSachenko) [inline docs/scos/dev/the-docker-sdk/202212.0/deploy-file/deploy-file-reference-1.0.md:144]: Starting a sentence with a variable name or value is not recommended. I would rephrase it as follows:
```suggestion
* `environment-configuration: secrets: grant:`—users' permissions. The `limited` permission provides read while `public` provides read and write permissions. The default one is `limited`.
```

File: docs/scos/dev/the-docker-sdk/202212.0/deploy-file/deploy-file-reference-1.0.md [hunk 5]
```diff
@@ -136,6 +136,27 @@ environment: 'docker'
...
+### environment-configuration: params:
+
+Defines parameters and their configuration:
+
+* `environment-configuration: params: name:`—parameter's name.
+* `environment-configuration: params: bucket:`—defines what the parameter  is used for. The acceptable value is `app`.
+* `environment-configuration: params: default:`—parameter's default value. Accepts string, number, and json values.
+* `environment-configuration: params: grant:`—IAM users' permissions. `limited` provides read while `public` provides read and write permissions. The default one is `limited`.
+
+***
```
Comment (VadymSachenko) [inline docs/scos/dev/the-docker-sdk/202212.0/deploy-file/deploy-file-reference-1.0.md:158]: Same
```suggestion
```

File: docs/cloud/dev/spryker-cloud-commerce-os/manage-maintenance-mode/enable-and-disable-maintenance-mode.md [hunk 6]
```diff
@@ -16,7 +16,7 @@ When you enable maintenance mode, the landing pages of your applications, like t
 2. On the **Pipelines** pane, click **Maintenance_Enable_{{ENVIRONMENT_NAME}}**.
 3. On the pipeline's page, click **Release change**.
 
-    The deployment time depends on the application's complexity. Once the deployment is finished, Storefront and Back Office visitors will see the maintenance page.
+    The deployment time depends on the application's complexity. Once the deployment is finished, Storefront and Back Office visitors will see the maintenance page. To get access to the frontend applications in maintenance mode, you can allowlist the needed IP addresses. For instructions, see [Configure access to applications in maintenance mode](/docs/cloud/dev/spryker-cloud-commerce-os/manage-maintenance-mode/configure-access-to-applications-in-maintenance-mode.html).
```
Comment (VadymSachenko) [inline docs/cloud/dev/spryker-cloud-commerce-os/manage-maintenance-mode/enable-and-disable-maintenance-mode.md:19]:
```suggestion
    The deployment time depends on the application's complexity. Once the deployment is finished, Storefront and Back Office visitors can see the maintenance page. To get access to the frontend applications in maintenance mode, you can allowlist the needed IP addresses. For instructions, see [Configure access to applications in maintenance mode](/docs/cloud/dev/spryker-cloud-commerce-os/manage-maintenance-mode/configure-access-to-applications-in-maintenance-mode.html).
```

**Review by VadymSachenko — CHANGES_REQUESTED** [review]
@andriitserkovnyi see my comments and suggestions
